### ISSUE TITLE
None working on Apple M1 Mac

### ISSUE THREAD

**Issue by easypickings** [sindresorhus/quick-look-plugins#126]
None of these plugins seems to work on M1 Mac, while they do work on my old intel Mac. Both Macs run Big Sur 11.2.3

**Comment by esaruoho** [comment]
I am seeing this same issue. Any chance of a M1-compatible list?

**Comment by easypickings** [comment]
> I am seeing this same issue. Any chance of a M1-compatible list?

well luckily I found `qlstephen`, `qlmarkdown` and `quicklook-json` do work though others still don't

**Comment by AndrejGajdos** [comment]
Would be good idea to add this information to Readme.

**Comment by ozzag** [comment]
Any news for Apple M1 devices ?

**Comment by alvarini** [comment]
Installed on M1 and not working :( even qlstephen that shows blank previews

**Comment by ibotu** [comment]
this needs more exposure! 🙏

**Comment by nijisakai** [comment]
seems no update yet

**Comment by mbaierl** [comment]
Just wanted to open the exact same report ...

**Comment by jspiro** [comment]
This is fallacious. Many work, not all. These are excellent:

Apparency, BetterZip, QLMarkdown (via sbarex), https://github.com/sbarex/SourceCodeSyntaxHighlight

**Comment by manuelgrabowski** [comment]
> well luckily I found qlstephen, qlmarkdown and quicklook-json do work though others still don't

Odd, `qlstephen` only shows a blank screen for me. Is it working for anyone else on an M1? I'm aware this is just a list and I'll create an issue in the proper project, but first wanted to check here amongst other M1 users for more reports.

**Comment by geersch** [comment]
> > I am seeing this same issue. Any chance of a M1-compatible list?
> 
> well luckily I found `qlstephen`, `qlmarkdown` and `quicklook-json` do work though others still don't

How did you get these to work? For me they don't work on Apple Silicon.

**Comment by easypickings** [comment]
> 

For what I’ve remembered it did work then. I gave up using them and turns to the app Peek though.

**Comment by PhilSwn** [comment]
> https://github.com/sbarex/SourceCodeSyntaxHighlight

Syntax Highlight is fantastic, customizable, well maintained, and works great on M1 and the newest MacOS (Currently Monterey 12.3.1) 

Thanks for posting it!

**Comment by canoejun** [comment]
<img width="1634" alt="image" src="https://user-images.githubusercontent.com/34930002/167599389-c9cdbf73-3dd9-42df-a5c9-88e88083b3e2.png">

it seems these plugins don't have correct authority , then cause I can't preview files

**Comment by PhilSwn** [comment]
@canoejun Yeah, qlcolorcode didn't work for me either. Try https://github.com/sbarex/SourceCodeSyntaxHighlight

**Comment by canoejun** [comment]
> @canoejun Yeah, qlcolorcode didn't work for me either. Try https://github.com/sbarex/SourceCodeSyntaxHighlight

I also try this, fortunatly, it's work great for me. you can try to copy "SyntaxHighlight" plugin in any folder.(suggest: the folder on desktop), then copy the plugin back to /Application folder for covering origin plugin.  For M1, it seems apple handle the software path in a different way.

**Comment by appindus-kanderson** [comment]
I had to relaunch finder for some of them to work.
https://9to5mac.com/2019/01/18/restart-finder-mac/
